src/Microsoft.DotNet.XHarness.CLI/Microsoft.DotNet.XHarness.CLI.csproj

```diff
@@ -46,5 +52,17 @@
       </Content>
     </ItemGroup>
   </Target>
-  
+
+  <Target Name="DownloadMlaunch" BeforeTargets="Build;Pack">
+    <Exec Condition="'$(IsWindows)' != 'true'" Command="chmod 0755 ../../eng/download-mlaunch.sh &amp;&amp; ../../eng/download-mlaunch.sh --commit $(MlaunchVersion) --target-dir $(IntermediateOutputPath)mlaunch" />
```

Comment:
I think you could just get rid of the OS logic above and do `Condition=" '$(OS)' == 'Windows_NT' "` and its opposite.